crates/zk-por-cli/src/prover.rs

```diff
@@ -24,30 +33,51 @@ use zk_por_core::{
 };
 use zk_por_tracing::{init_tracing, TraceConfig};
 
+// as we use one thread to prove each batch, we load num_cpus batches to increase the parallelism.
+fn calculate_per_parse_account_num(batch_size: usize) -> usize {
+    let num_cpus = num_cpus::get();
+    let num_cpus =
+        if BATCH_PROVING_THREADS_NUM < num_cpus { BATCH_PROVING_THREADS_NUM } else { num_cpus };
+    num_cpus * batch_size
+}
+
 pub fn prove(cfg: ProverConfig, proof_output_path: PathBuf) -> Result<(), PoRError> {
-    let trace_cfg: TraceConfig = cfg.log.into();
-    let _g = init_tracing(trace_cfg);
+    let trace_cfg: TraceConfig = cfg
+        .log
+        .unwrap_or(ConfigLog {
```

Comment:
it's very common to provide this config to user; we treat our software as open source software, not only a tool to be used by none developper